MANIFEST.in

```
@@ -1,2 +1,5 @@
 include requirements.txt
-include CHANGELOG.rst
\ No newline at end of file
+include CHANGELOG.rst
+include lightning_qubit.toml
+include lightning_kokkos.toml
+include lightning_gpu.toml
```

Unless these come in alphabetical order, we use the canonical order qubit < gpu < kokkos. 
```suggestion
include lightning_qubit.toml
include lightning_gpu.toml
include lightning_kokkos.toml
```